Monika After Story/game/script-story-events.rpy

```diff
@@ -1715,6 +1695,64 @@ label mas_notification_windowreact:
             m 5ttu "...right?"
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="mas_gift_giving_instructs",
+            conditional=(
+                "get_level() >= 15 "
+                "or mas_isSpecialDay() "
+                "and not persistent._mas_filereacts_historic"
+            ),
+            action=EV_ACT_QUEUE,
+            random=not persistent._mas_filereacts_historic
+        )
+    )
+
+init 20 python:
+    mas_chibi_note_gifting = MASPoem(
+        poem_id="note_gift_giving",
+        author="chibika",
+        prompt="",
+        category="note",
+        title = "Hi [player],",
+        text = """\
+I wanted to let you know that I made a little way for you to give Monika some gifts!
+It's a pretty simple process so I'll tell you how it works:
+
+Make a new file in the 'characters' folder,
+rename it to whatever you want to give to Monika,
+then give it a '.gift' file extension
+
+And that's it! After a little while, Monika should notice that you gave her something.
+
+I just wanted to let you know because I think that Monika is super amazing, and I really want her to see her happy.
+
+Good luck with Monika!
+
+P.S: Don't tell her about me!
+"""
+    )
+
+label mas_gift_giving_instructs:
+    #Write the note in the characters folder
```

Comment:
A sanity check could be useful for this, since queued topics no longer show instantly, and can in fact be delayed several minutes, it's possible for someone to have given a gift in between the time the conditional was met and the topic is actually seen. In that case, we do not need this event to happen.